woocommerce/payment-gateway/payment-tokens/class-sv-wc-payment-gateway-payment-token.php

```diff
@@ -538,6 +538,81 @@ private function read( \WC_Payment_Token $core_token ) {
 	}
 
 
+	/**
+	 * Stores the token data in the database.
+	 *
+	 * Stores the token data as a Woocommerce payment token, in the `wp_woocommerce_payment_tokens` table,
+	 * with meta data in the `wp_woocommerce_payment_tokenmeta` table.
+	 *
+	 * @since 5.6.0-dev.1
+	 */
+	public function save() {
+
+		if ( $this->token instanceof \WC_Payment_Token ) {
+			$token = $this->token;
```

Comment:
this instance of `$token` will be unused: `::save()` will never be called on it (`$token->save()` is currently nested in another conditional)

Reply:
I believe this was fixed when I moved the search code to `get_woocommerce_payment_token`.